### ISSUE TITLE
Trust interstitial for tunnels doesn't appear authentic

### ISSUE THREAD

**Issue by clocky** [microsoft/vscode#176276]
Does this issue occur when all extensions are disabled?: No (requires Tunnel extension)

- VS Code Version: 1.76.0 
- OS Version: macOS 13.2.1

TL;DR, the page where users are asked to trust a remote tunnel, doesn't feel like it's an authentic VSCode or Microsoft page at all.

Steps to Reproduce:

1. Connect to a tunnel
2. Approve authentication method via GitHub
3. An interstitial screen appears asking to confirm if the user believes that the link to the tunnel is trustworthy, with options to continue or report an unsafe page.
4. The screen doesn't feel authentic; it uses unusual type choices and colours for the buttons that are off brand for Microsoft, and the "Copyright Microsoft 2022" footer doesn't engender trust with a missing copyright symbol and the incorrect year.

![image](https://user-images.githubusercontent.com/1550587/223192059-19c61069-1e9d-4a0f-9d69-7549b2568d97.png)

![image](https://user-images.githubusercontent.com/1550587/223192115-c8ad2f58-ed10-416b-a172-7e3d59d75bc0.png)

/cc @daviddossett

**Comment by daviddossett** [comment]
First I'm seeing this and this doesn't look like something we've built. @aeschli do you have any context here?

**Comment by clocky** [comment]
I realised the steps above aren't quite correct (the outcome remains the same). My specific steps were:

1. Connect to tunneled instance of VSCode on another machine.
2. Open up a node project on the remote machine.
3. Start a terminal
4. Run `npm run start` that starts a local server at `https://localhost:8080/`
5. A warning toast pop-up notes in local VSCode instance that `localhost:8080` will be tunneled.
6. Cmd-click `localhost:8080` in the terminal output
7. GitHub authentication screen pops up in a new browser window.
8. Approve use of GitHub credentials.
9. **Then the screen above is shown.**
10. Hit continue, view tunneled localhost as normal.

**Comment by connor4312** [comment]
This is on the Dev Tunnels side, @derekbekoe

**Comment by derekbekoe** [comment]
Thanks for the feedback - We'll get the copyright year updated and take a look at reviewing the design of the page.

**Comment by connor4312** [comment]
I'll go ahead and close this out since this is now tracked internally in Dev Tunnels. Thanks for the issue 🙂
